proof_of_concept/asset_store.py

```diff
@@ -53,17 +53,19 @@ def retrieve(self, asset_id: str, requester: str
             KeyError: If no asset with the given id is stored here.
 
         """
-        logger.info(f'{self} servicing request from {requester} for data: '
-                    f'{asset_id}, ')
+        logger.info(f'{self}: servicing request from {requester} for data: '
```

Comment:
Can we discuss f-strings and support for older Python versions?

Reply:
We discussed that we will stick to f-strings for now, and therefore not support older python versions.